source/apiVolontaria/apiVolontaria/urls.py

```diff
@@ -30,6 +30,11 @@
         ObtainTemporaryAuthToken.as_view(),
         name='token_api'
     ),
+    # Volunteer
+    url(
+        r'^volunteer/',
+        include('volunteer.urls', namespace="volunteer")
```

Comment:
coma at end of the line

Reply:
I'll fix the other url patterns then. I'll add a comma each